README.md

```diff
@@ -23,18 +23,14 @@ int main() {
 - Compatible with old(er) compiler versions; at least `gcc` versions => `4.8` & `clang` => `5.0.0` (previous 
 versions have not been checked, so I'd say at least a compiler with C++11 support).
 - Tested with `-Wpedantic -Wextra -Wall -Wno-unused-function` and `/W4` for MSVC
-- One dependency ([`fmt`](https://github.com/fmtlib/fmt)) which is automatically configured, or none if you use `#define LZ_STANDALONE` before including or use CMake `-D CPP-LAZY_USE_STANDALONE`
+- One dependency ([`{fmt}`](https://github.com/fmtlib/fmt)) which is automatically configured, or none if you use `#define LZ_STANDALONE` before including or use CMake `-D CPP-LAZY_USE_STANDALONE`

```

Comment:
Refer to installation maybe

Reply:
I do not agree since this is just a list of features. It is already refered to before when `{fmt}` is mentioned for the first time.

Reply:
Then "One dependency `{fmt}`" will suffice